setup-client.yml

```diff
@@ -138,9 +138,23 @@
       dest: /etc/gdm3/greeter.dconf-defaults
     register: gdm_file
     tags: gdm
+  - name: copy ubuntu.css to client
+    copy:
+      src: ubuntu-adfc.css
+      dest: /usr/share/gnome-shell/theme/ubuntu.css
```

Comment:
Existiert die Datei schon? Dann finde ich das schon etwas kritisch. Was ist z.B. wenn ein Ubuntu-Update  die Datei überschreibt?

Reply:
Ja, die Datei existiert. 
Im Zweifelsfall müsste nach einem Update, das die Datei überschreiben würde, das entsprechende Playbook erneut ausgeführt werden (-t gdm)

Reply:
Und stattdessen ein neues Theme erstellen? Keine Ahnung wie aufwändig das wäre....

Reply:
hmm. 
"Creating a Plasma Theme in 7 Easy Steps"
https://techbase.kde.org/Development/Tutorials/Plasma4/Theme

Ich werde es mir mal anschauen.